xtask/src/swift.rs

```diff
@@ -276,6 +253,81 @@ fn build_xcframework(
     Ok(())
 }
 
+/// Builds the SDK for the specified targets and profile.
+fn build_targets(
+    targets: Vec<&Target>,
+    profile: &str,
+    sequentially: bool,
+) -> Result<HashMap<Platform, Vec<Utf8PathBuf>>> {
+    if sequentially {
```

Comment:
We can get rid of this if we go with only building for iOS and simulators.

Reply:
Along with this.